docs/src/comparison.md

```diff
@@ -16,14 +16,14 @@ For LOC, we use the following convention: code is formatted using standard pract
 
 | Model/Framework | Agents | Mesa | Netlogo | MASON |
 |---|---|---|---|---|
-|Wolf Sheep Grass|1|7.1x|2.1x|NA|
-|(LOC)|139|273|137 (871)| . |
+|Wolf Sheep Grass|1|6.9x|2.1x|NA|
+|(LOC)|139|238|137 (871)| . |
 |Flocking|1|29.7x|10.3xᕯ|2.1x|
 |(LOC)|66|120|82 (689)|369|
-|Forest Fire|1|29.1x|4.1x|NA|
-|(LOC)|27|61|43 (545)|.|
-|Schelling|1|31.5x|8.0x|14.3x|
-|(LOC)|34|63|68 (732)|248|
+|Forest Fire|1|22.5x|4.1x|NA|
+|(LOC)|27|42|43 (545)|.|
```

Comment:
I counted the lines for Mesa by hand three times (ignoring docstrings, comment, empty lines). I got 31. How is the count generated?

I also counted the lines for forestfire.jl, got 26.

Reply:
Oh, I forgot to say that I removed
```python
def get_pos(self):
    return self.pos
```
from ForestFire/model.py, which resulted in 2 lines reduction.

Reply:
With https://github.com/JuliaDynamics/Agents.jl/pull/394/commits/1b3651aa3e83933209d63aa56bfe1bf3a96cf5d5 it is down to 28.

Reply:
The commit above is dead because I reworded the commit message. For a more permanent reference, it is the 4th commit of https://github.com/JuliaDynamics/Agents.jl/pull/394/commits.

Reply:
Cool, I'll comment over on your new PR.